### ISSUE TITLE
Allow a simple 'unnumbered: True' attribute on links

### ISSUE THREAD

**Issue by ipspace** [ipspace/netsim-tools#92]
### Discussed in https://github.com/ipspace/netsim-tools/discussions/91

<div type='discussions-op-text'>

<sup>Originally posted by **jbemmel** December 27, 2021</sup>
https://github.com/ipspace/netsim-tools/blob/master/docs/addressing.md#unnumbered-interface-support works, but is more complicated than it needs to be imho

Instead of:
```
role: <some-custom-pool-with-unnumbered-flag-set>
```

why not simply:
```
unnumbered: True
```</div>

**Comment by ipspace** [comment]
To be feature-complete, we need a protocol-level (IPv4/IPv6) support for unnumbered interfaces. We could have a numbered IPv4 + LLA IPv6, or unnumbered IPv4 + configured IPv6.

**Comment by ipspace** [comment]
Implemented as part of #97
